### PR TITLE
fix(Core/MiscHandler): Dungeon requirements Message

### PR DESCRIPTION
**Pull request by pvp100com**
## Changes Proposed:
-  Dungeon requirements Level and Item, SendAreaTriggerMessage variable %s, %u  error

## Issues Addressed:

- fix dungeon access requirements message error
- fix level variable always 0 and item name get locale not work

## SOURCE:

- Original:

- Fixed:

## Tests Performed:

- It builds without errors
- Tested in-game

- Server:
AzerothCore rev. https://github.com/azerothcore/azerothcore-wotlk/commit/893a9beeacc4554f9ac85755b40520a9b8e6d411+
Debian 11 x64
MySQL 8.0.33

- Client:
3.3.5a.12340 zhCN
Windows 10 x64 LTSC

## How to Test the Changes:

1. SELECT * FROM acore_string WHERE entry = 50;
2. check "content_default" AND "locale_zhCN" value
3. in-game, and .go xyz  -301.8008 3162.2664 31.720385 530
4. Try to enter the instance, observe message.

### CODE REVIEW

**Review by Helias — APPROVED** [review]

File: src/server/game/Battlefield/Zones/BattlefieldWG.h [hunk 1]
```diff
@@ -1041,7 +1041,7 @@ const WintergraspTowerCannonData TowerCannon[WG_MAX_TOWER_CANNON] =
         2,
         {
             { 4448.138184f, 1974.998779f, 441.995911f, 1.967238f },
-            { 4448.713379f, 1955.148682f, 441.995178f, 0.380733f },
+            { 4489.819824f, 1955.636108f, 441.996002f, 0.353242f },
```
Comment (Helias) [inline src/server/game/Battlefield/Zones/BattlefieldWG.h:1044]: why it was necessary?

Comment (Kitzunu) [reply src/server/game/Battlefield/Zones/BattlefieldWG.h:1044]: Revert this. It has nothing to do with the PR

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: src/server/game/Battlefield/Zones/BattlefieldWG.h (hunk 1 above)
Comment (Helias) [inline src/server/game/Battlefield/Zones/BattlefieldWG.h:1044]:
```suggestion
            { 4448.713379f, 1955.148682f, 441.995178f, 0.380733f },
```

Comment (Helias) [reply src/server/game/Battlefield/Zones/BattlefieldWG.h]: reverted

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
@azerothcore/testers can someone with a locale client test it

**Pull request merged**